docs/README.md

```diff
@@ -5,14 +5,58 @@ This web application is built as a [Jekyll](https://jekyllrb.com/) site and is i
 ### Charts
 
 Each `.html` page in this directory contains one or more charts.
-Each chart is defined as an `<svg>` element with a few special attributes that define their behavior:
+Charts are created from `<div>` containers of the class `chart-placeholder` by [`charts.js`](assets/js/charts.js).
```

Comment:
Perhaps we can word this as "Charts are dynamically created by [`charts.js`](assets/js/charts.js) from `<div class="chart-placeholder">` placeholders present in the HTML" ?

Reply:
Thanks, I’ll incorporate your feedback!